src/main/java/io/confluent/idesidecar/restapi/credentials/Redactable.java

```diff
@@ -68,7 +68,7 @@ public final char[] asCharArray() {
    */
   @JsonIgnore
   public final String asString() {
-    return new String(this.asCharArray());
+    return new String(raw);
```

Comment:
The `this.asCharArray()` makes a copy of the raw array; then String makes another copy. This change avoids the extra copy.